packages/react-reconciler/src/ReactFiberCompleteWork.new.js

```diff
@@ -1593,11 +1595,10 @@ function completeWork(
           // Run passive effects to retain/release the cache.
           workInProgress.flags |= Passive;
         }
-        if (current !== null) {
-          popTransition(workInProgress);
-        }
       }
 
+      popTransition(workInProgress, current);
```

Comment:
This isn't wrapped in a feature flag check anymore but it should be

Reply:
I guess it's fine because the body of the function already is wrapped in feature flags